inc/namespace.php

```diff
@@ -146,7 +146,7 @@ function set_wp_privacy_exports_dir( string $dir ) {
 	if ( strpos( $dir, 's3://' ) !== 0 ) {
 		return $dir;
 	}
-	$dir = sys_get_temp_dir() . '/wp_privacy_exports_dir/';
+	$dir = get_temp_dir() . '/wp_privacy_exports_dir/';
```

Comment:
```
wp> sys_get_temp_dir() 
=> string(4) "/tmp"
wp> get_temp_dir() 
=> string(5) "/tmp/"
```

`get_temp_dir` adds a trailing slash, though as far as I know PHP treats it as a single slash

Reply:
That's true. Thank you for pointing that out. I've changed that line now.